version: '3.9'

networks:
  internal:
  external:

services:
  scv-asssess-rhiinitis:
    image: va/abd_vro-rhinitis:latest
    build: ../../../service-python/rhinitis/src/.
    environment:
      RABBITMQ_PORT: 5672
      RABBITMQ_RETRY_LIMIT: 3
      RABBITMQ_TIMEOUT: 10800
      RABBITMQ_PLACEHOLDERS_HOST: rabbitmq-service
      RABBITMQ_PLACEHOLDERS_USERNAME: ${RABBITMQ_PLACEHOLDERS_USERNAME}
      RABBITMQ_PLACEHOLDERS_USERPASSWORD: ${RABBITMQ_PLACEHOLDERS_USERPASSWORD}
    security_opt:
      - no-new-privileges:true
    depends_on:
      rabbitmq-service:
        condition: service_healthy
    networks:
      - internal
    restart: on-failure:5
